pnpm-lock.yaml

```diff
@@ -380,11 +380,34 @@ packages:
       - supports-color
     dev: true
 
-  /@astrojs/mdx/0.17.2_astro@2.0.17:
+  /@astrojs/markdown-remark/2.1.1_astro@2.1.4:
```

Comment:
Hmm, how come there's two versions of `@astrojs/markdown-remark`?

Reply:
Hmm. Maybe we bumped in `astro` but we’re not on the latest of the MDX integration? 🤔

Reply:
Yup, bumping MDX to latest moves us back to just one copy of markdown-remark.